modules/iam/main.tf

```diff
@@ -125,6 +124,36 @@ data "aws_iam_policy_document" "segment_data_lake_policy_document" {
     effect = "Allow"
   }
 
+  # Explicitly deny Segment to modify IAM or sensible configuration from the Data Lake S3 bucket.
+  statement {
```

Comment:
Is it OK to deny these actions?

Reply:
Yes we do not make any of these API calls so denying these sounds good 👍